### PR TITLE
Edit User Guide to match updates in ConcertAddressBook

### LINKED ISSUE #75: Update guides
- update User Guide with new commands
- update Developer Guide with new commands and any new diagrams

### PR DESCRIPTION
**Pull request by SKarthikeyan28**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,23 +2,33 @@
...
+5. [Command Summary](#command-summary)
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick start
 
 1. Ensure you have Java `17` or above installed in your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+1. Download the latest `.jar` file from [here](nil). (COMING SOON)
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:27]: use our repo link for the releases page

File: docs/UserGuide.md [hunk 2]
```diff
@@ -2,23 +2,33 @@
...
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your ConcertAddressBook.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar tp.jar` command to run the application.<br>
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:31]: go to build.gradle to change the name from addressbook to concertphonebook `line 69` then test with `./gradlew clean shadowjar` and check inside `build/libs` for the jar file should be changed to the new name

File: docs/UserGuide.md [hunk 3]
```diff
@@ -27,14 +37,21 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `addp n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 r/organiser t/friends` : Adds a contact named `John Doe` to the Address Book.
 
    * `delete 3` : Deletes the 3rd contact shown in the current list.
 
    * `clear` : Deletes all contacts.
-
+   
+   * `addc` : Adds Concert to the ConcertPhoneBook
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:46]: can include the sample add concert format follow the add-person

File: docs/UserGuide.md [hunk 4]
```diff
@@ -27,14 +37,21 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+   
+   * `addcc` : Links a person to a concert
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:48]: same here

File: docs/UserGuide.md [hunk 5]
```diff
@@ -27,14 +37,21 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+   
+   * `findp` : Finds person from list of contacts
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:50]: this too

File: docs/UserGuide.md [hunk 6]
```diff
@@ -27,14 +37,21 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `addp n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 r/organiser t/friends` : Adds a contact named `John Doe` to the Address Book.
 
    * `delete 3` : Deletes the 3rd contact shown in the current list.
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:42]: there is deletep deletec and deletecc

File: docs/UserGuide.md [hunk 7]
```diff
@@ -63,40 +80,57 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a person: `addp`
 
-Adds a person to the address book.
+Adds a person to the Concert Address book.
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:96]: Concert phone book or choose one need to be consistent with the above stuff

File: docs/UserGuide.md [hunk 8]
```diff
@@ -109,27 +143,25 @@ Examples:
...
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `findp n/Alex Roy r/Artist` returns `Alex Yu and Roy Balakrishnan` <br>
+  ![result for 'find alex roy'](images/findAlexRoyResult.png)
 
 ### Deleting a person : `delete`
 
-Deletes the specified person from the address book.
+Deletes the specified person from the Concert address book.
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:164]: this command is now deletep need to add for deletec and deletecc

File: docs/UserGuide.md [hunk 9]
```diff
@@ -155,20 +187,19 @@ Format: `exit`
...
+ConcertAddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+ConcertAddressBook data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
+If your changes to the data file makes its format invalid, ConcertAddressBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:197]: here also either concert phone book for all or concert address book

File: docs/UserGuide.md [hunk 10]
```diff
@@ -63,40 +80,57 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+Format: `addc n/CONCERTNAME a/ADDRESS d/DATE`
+
+Examples:
+* `addc n/Coachella a/81800 51st Ave, Indio, Southern California, United States d/2024-12-20 1010`
+
+### Adding a ConcertContact: `addcc`
+
+Adds an association to the contact in the Concert Address book with another Concert. 
+
+Format: `addcc n/CONCERTNAME a/ADDRESS d/DATE`
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:121]: wrong format go check the updated one

File: docs/UserGuide.md [hunk 11]
```diff
@@ -63,40 +80,57 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `addp n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/Organiser`
+* `addp n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison r/Artist p/1234567 t/criminal`
+
+### Adding a Concert: `addc`
+
+Adds a Concert to the Concert Address book.
+
+Format: `addc n/CONCERTNAME a/ADDRESS d/DATE`
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:112]: CONCERT_NAME

File: docs/UserGuide.md [hunk 12]
```diff
@@ -109,27 +143,25 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
-### Locating persons by name: `find`
+### Finding a person: `findp`
 
 Finds persons whose names contain any of the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `findp n/KEYWORD [KEYWORD] r/ROLE`
```
Comment (wes-w-z-h) [inline docs/UserGuide.md:150]: Wrong format check the updated one

**Review by wes-w-z-h — CHANGES_REQUESTED** [review]
Quite a lot of changes try to finish them asap

**Review by bbbbcccd — APPROVED** [review]
lgtm

**Review by wes-w-z-h — APPROVED** [review]

**Pull request closed without merging**